src/output/icons.rs

```diff
@@ -670,6 +701,7 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "tzo"            => Icons::COMPRESSED,       // 
     "unity"          => Icons::UNITY,            // 
     "unity3d"        => Icons::UNITY,            // 
+    "url"            => '\u{f0ac}',              // 
```

Comment:
Seems like \u{f1fa} could be a better choice.

Reply:
I'm not sure why, the "@" icon seems specific to email for some reason. Maybe someone else can weigh in on this one.